src/js/hooks/cycleCount/useResolveStepTable.jsx

```diff
@@ -272,15 +278,21 @@ const useResolveStepTable = ({
         addNewRow: () => addEmptyRow(productCode, cycleCountId),
         isNewRow,
       });
-
+      const isWiderWidth = [cycleCountColumn.ROOT_CAUSE, cycleCountColumn.COMMENT].includes(id);
```

Comment:
the same third question :p

Reply:
Why it cannot be just width auto + proper flexWidth set in meta?